manufacturing-ai-suite/industrial-edge-insights-time-series/apps/weld-anomaly-detection/time-series-analytics-config/udfs/weld_anomaly_detector.py

```diff
@@ -4,23 +4,21 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-""" Custom user defined function for anomaly detection on 
-the weld data. """
+""" Custom user defined function for anomaly detection for weld_defect_detection. """
```

Comment:
Let's rename on the below lines:
Custom user defined function for anomaly detection in weld sensor data.